### PR TITLE
Rclone support

### PR DESCRIPTION
**Pull request by jaapmarcus**
Setup:
- Install Rclone 
```
curl https://rclone.org/install.sh | sudo bash
```
- Run rclone config 
- Follow the steps
- Go to Settings -> Backups  -> Select "Rclone" enter "Host" for example "b2" and folder / bucket / what ever is required: "hestiacp" for example

### CODE REVIEW

**Comment by jaapmarcus** [comment]
It will replace current B2 implementation and also delete the old Google Cloud code

**Review by tlcd96 — COMMENTED** [review]
After reviewing the files, it appears that all is ok, but i have to say that i don't know rclone commands so this review can be totally ok or I haven't spot any bug that might exist.

File: web/templates/pages/edit_server.html [hunk 1]
```diff
@@ -825,6 +826,33 @@
...
+															<tr>
+																<td class="vst-text">
+																	<?=_('Host');?>
+																</td>
+															</tr>
+															<tr>
+																<td>
+																	<input type="text" size="20" class="vst-input" name="v_rclone_host" value="<?=trim($v_rclone_host, "'")?>">
+																</td>
+															</tr>
```
Comment (AlecRust) [inline web/templates/pages/edit_server.html]: This is the improved style I've been migrating to, should look the same:
```suggestion
															<tr>
																<td class="vst-text">
																	<label for="v_rclone_host" class="form-label"><?=_('Host');?></label>
																	<input type="text" class="form-control" name="v_rclone_host" id="v_rclone_host" value="<?=trim($v_rclone_host, "'")?>">
																</td>
															</tr>
```

**Comment by jaapmarcus** [comment]
👍🏼

**Review by ScIT-Raphael — APPROVED** [review]

**Comment by darkworks** [comment]
Rclone support missing in docs , need to be added to docs

**Comment by jaapmarcus** [comment]
Current docs is getting replaced with 

https://development.hestiacp.com/docs/server-administration/backup-restore#how-to-setup-rclone

**Pull request merged**